.github/workflows/linters.yml

```diff
@@ -6,20 +6,32 @@ env:
   FORCE_COLOR: 1
 
 jobs:
-  eslint:
-    name: ESLint
+  lighthouse:
+    name: Lighthouse
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
           node-version: "18.x"
```

Comment:
Hey @yin-ka change this line to :
```
node-version: "12.x"
```

Reply:
@didierganthier thank you. It has been resolved.